index.html

```diff
@@ -48,10 +51,9 @@ <h2>let's connect</h2>
       <a href="#" class="five"><img src="./images/Vector.png" alt=""></a>
      </nav>
```

Comment:
- [ ] Kindly make sure the list of icons is wrap in a `ul` tag, and `li` tag per icon; each icon has a link (`a` tag).
  * Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.